spacify-portal/src/main/sql/important-queries.sql

```diff
@@ -123,3 +123,11 @@ alter table corespacify.incentive
             references corespacify.user
 
 -- Now insert mock data into this using the Insert queries in mock-data.sql. You do not need to specify primary key in inserts. DB will handle that.
+
```

Comment:
lets run this query on the common database too.

Reply:
done!